### PR TITLE
Small fix to check in document Click if the user is clicking the scrollbar

### PR DESCRIPTION
**Pull request by oizie**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -400,6 +400,10 @@ const Trigger = createReactClass({
     return childPros[event] || props[event];
   },
 
+  isClickedOnScrollBar(clientX) {
+    return clientX >= document.documentElement.offsetWidth;
```
Comment (benjycui) [inline src/index.js]: You should check horizontal scrollbar, too.

File: src/index.js [hunk 2]
```diff
@@ -400,6 +400,11 @@ const Trigger = createReactClass({
     return childPros[event] || props[event];
   },
 
+  isClickedOnScrollBar(clientX) {
+    return clientX >= document.documentElement.offsetWidth ||
+      event.clientY >= document.documentElement.clientHeight;
```
Comment (benjycui) [inline src/index.js]: You didn't try to test this locally? 👎

Comment (benjycui) [inline src/index.js]: You should link your modified rc-trigger in rc-tooltip, and test if your modification works before PR, thanks.

**Comment by oizie** [comment]
Ei, yes I did test it. That's way I am pushing.

**Comment by oizie** [comment]
If you want I can create some tests to test this. I just didn't have much time for that.

File: src/index.js (hunk 2 above)
Comment (benjycui) [inline src/index.js]: `event` is not defined.

**Comment by benjycui** [comment]
https://github.com/react-component/trigger/pull/60/files#r122657602

**Comment by oizie** [comment]
Sorry I am using the web editor because the proxy in the company doesn't allow me to push to Github. :/

**Comment by oizie** [comment]
Sorry is this ok to be merged? or do you further changes?

File: src/index.js [hunk 3]
```diff
@@ -400,6 +402,11 @@ const Trigger = createReactClass({
     return childPros[event] || props[event];
   },
 
+  isClickedOnScrollBar(clientX, clientY) {
+    return clientX >= document.documentElement.offsetWidth ||
```
Comment (benjycui) [inline src/index.js]: Actually, you code doesn't work as expect. Please test this before PR.

**Comment by benjycui** [comment]
Sorry, you PR doesn't fix the issue, close now.

**Comment by oizie** [comment]
It doesnt? Strange I was using internally and was fine. Was about to fix the limiting issues so it would pass the build. If you have a better idea to do it work away.

**Comment by benjycui** [comment]
Try your PR in Mac Chrome, and some pages which in iframe.

**Pull request closed without merging**